### PR TITLE
WIP XML Schema

### PR DESCRIPTION
**Pull request by andrewbiang888**

### CODE REVIEW

File: api/models/converters/index.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import OrestarContributionConverter, { ContributionConvertType } from './orestarContributionConverter';
+import OrestarExpenditureConverter, { ExpenditureConvertType } from './orestarExpenditureConverter';
+
+// import { Expenditure } from '../entity/Expenditure';
```
Comment (danmelton) [inline api/models/converters/index.ts]: can we remove if commented out

File: api/models/entity/Contribution.ts [hunk 2]
```diff
@@ -103,7 +105,8 @@ export enum PaymentMethod {
     MONEY_ORDER = 'money_order',
     CREDIT_CARD_ONLINE = 'credit_card_online',
     CREDIT_CARD_PAPER = 'credit_card_paper',
-    ETF = 'electronic_funds_transfer'
+    ETF = 'electronic_funds_transfer',
+    DEBIT = 'debit'
```
Comment (danmelton) [inline api/models/entity/Contribution.ts:109]: this needs to get added to the front end too

File: api/test/factories.ts [hunk 3]
```diff
@@ -81,7 +85,7 @@ export async function newContributionAsync(campaign: Campaign, government: Gover
         contribution.state = 'OR';
         contribution.status = ContributionStatus.DRAFT;
         contribution.zip = '97214';
-        contribution.paymentMethod = PaymentMethod.CASH;
+        contribution.paymentMethod = PaymentMethod.CHECK;
```
Comment (danmelton) [inline api/test/factories.ts:88]: if its a check, it need a check number too

File: app/src/Pages/Portal/Expenses/ExpendituresFields.js [hunk 4]
```diff
@@ -228,7 +228,8 @@ export const fields = {
     component: SelectField,
     options: {
       values: [
-        { value: PaymentMethodEnum.CASH, label: 'Cash' },
+        // { value: PaymentMethodEnum.CASH, label: 'Cash' },
```
Comment (jaronheard) [inline app/src/Pages/Portal/Expenses/ExpendituresFields.js]: Can we add a comment about why we commented this out?

File: api/models/entity/Contribution.ts [hunk 5]
```diff
@@ -680,6 +684,7 @@ export async function getContributionsByGovernmentIdAsync(
         });
 
         const total = await contributionRepository.count(removeUndefined({ where }));
+        console.log({ options });
```
Comment (jaronheard) [inline api/models/entity/Contribution.ts]: Should this log still be there?

**Review by jaronheard — APPROVED** [review]

**Pull request merged**